README.md

```diff
@@ -13,7 +13,7 @@ This package provides a Python interface for generating coordinate tables and re
 
 If you are using `atlasreader` in your publication, please cite the following paper:
 
-Markello R., Gale D., Herholz P., Whitaker K., & Notter M. P. (2019). AtlasReader: Quick generation of coordinate tables and region labels for statistical MRI images. *The Journal of Open Source Software*, ZZZ, YYY.
+Markello R., Gale D., Herholz P., Whitaker K., Notter-Bielser M.-L., & Notter M. P. (2019). AtlasReader: A Python package to generate coordinate tables, region labels and informative figures from statistical MRI images. *The Journal of Open Source Software*, ZZZ, YYY.
```

Comment:
@miykael @kirstiejane: tagging you both here because my suggested re-ordering would affect you. As I said, though, other are welcome to discuss this, too! I know this topic is everyone's least favorite aspect of, well, everything, so apologies for raising it... Nevertheless, I feel quite strongly about @miykael's role in this project!

Reply:
@rmarkello - I'm ok with this change. I thought that this might give me a last author ship paper, but I'm also very happy to have my name as "et al." :-) So for me this is fine.

And that @KirstieJane is last, is clear (and she will surely argue against it). But it was her at the Hackathon in Lausanne who inspired me with the "Brains4Publication". And after asking "hey, I have some code to visualize some things", she said: "Yes! Let's go for it!"

So +1 for having me first and Kirstie last author.

Reply:
Bahahahaha - I had missed this last part of your reply @miykael and was *literally* just writing a paragraph about how I shouldn't be last 😂 🤣 😂.

I'm happy with this if everyone else is. I had had conversations with Michael about him being last, but I agree with @rmarkello that the main credit should go to Michael and so having his name first and everyone else's alphabetical makes sense to me.

Hugs to everyone 🤗 It's a wonderful project.